org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/GetRefactorEditHandler.java

```diff
@@ -58,6 +61,7 @@ public static RefactorWorkspaceEdit getEditsForRefactor(GetRefactorEditParams pa
 		IProblemLocationCore[] locations = CodeActionHandler.getProblemLocationCores(unit, params.context.getContext().getDiagnostics());
 		boolean problemsAtLocation = locations.length != 0;
 		String positionKey = DEFAULT_POSITION_KEY;
+		Command additionalCommand = null;
```

Comment:
unnecessary change. only declare the variable when you need it. here is too early.

Reply:
Fixed.